guides/md/keras_cv/retina_net_overview.md

```diff
@@ -124,57 +118,6 @@ train_ds = train_ds.map(unpackage_tfds_inputs, num_parallel_calls=tf.data.AUTOTU
 eval_ds = eval_ds.map(unpackage_tfds_inputs, num_parallel_calls=tf.data.AUTOTUNE)
 ```
 
-<div class="k-default-codeblock">
-```
-[1mDownloading and preparing dataset 868.85 MiB (download: 868.85 MiB, generated: Unknown size, total: 868.85 MiB) to ~/tensorflow_datasets/voc/2007/4.0.0...[0m
```

Comment:
Please manually remove unnecessary logs (it harm readability)

Reply:
Done

Reply:
But not pushed?

Reply:
this is a removal not an addition, right?

Reply:
it was in the guide last render - but I removed in this PR